### PR TITLE
Lucead Bid Adapter: Update

### PR DESCRIPTION
**Pull request by aymeric45**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter  
- [x] Updated bidder adapter   
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
- Support Single Request Architecture mode
- Enhanced reporting
- Updated PAAPI auction config
- Added region parameter

## Other information
Docs update PR: https://github.com/prebid/prebid.github.io/pull/5319

### CODE REVIEW

File: modules/luceadBidAdapter.js [hunk 1]
```diff
@@ -6,16 +6,18 @@ import {fetch} from '../src/ajax.js';
...
-let endpointUrl = 'https://prebid.lucead.com/go';
 const defaultCurrency = 'EUR';
 const defaultTtl = 500;
 const aliases = ['adliveplus'];
+const defaultRegion = 'eu';
+const domain = 'lucead.com'
+let baseUrl = `https://${domain}`;
+let staticUrl = `https://s.${domain}`;
+let endpointUrl = baseUrl;
+let companionUrl = 'https://cdn.jsdelivr.net/gh/lucead/prebid-js-external-js-lucead@master/dist/prod.min.js';
```
Comment (patmmccann) [inline modules/luceadBidAdapter.js:17]: this is not allowed, you have to delete this. See #10653

**Review by patmmccann — CHANGES_REQUESTED** [review]

**Comment by aymeric45** [comment]
Hello
Thanks for your feedback.
We are currently using the feature described here https://github.com/prebid/prebid-js-external-js-template and are fully compliant with the policy.
The source code can be audited here : https://github.com/lucead/prebid-js-external-js-lucead/tree/master/src
The build process can be found here: https://github.com/lucead/prebid-js-external-js-lucead/blob/master/package.json
Are there changes to be made to be compliant with PB9 while still using `loadExternalScript` ?

**Comment by ChrisHuie** [comment]
Doesn't appear you are passing schain. Please add as all adapters are now expected to support schain.

File: modules/luceadBidAdapter.js (hunk 1 above)
Comment (aymeric45) [reply modules/luceadBidAdapter.js]: removed.

File: modules/luceadRtdProvider.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  return location.hash.includes('prebid-dev');
+}
+
+export function init() {
+  if (isDevEnv()) {
+    companionUrl = '/dist/prebid-companion.js';
+  }
+
+  window.lucead_prebid_load_promise = new Promise(resolve => {
+    loadExternalScript(companionUrl, name, () => {
```
Comment (patmmccann) [inline modules/luceadRtdProvider.js:27]: This just bids in a sidecar to prebid, it isn't appropriate for our project

**Review by patmmccann — CHANGES_REQUESTED** [review]

File: modules/luceadBidAdapter.js [hunk 3]
```diff
@@ -1,40 +1,43 @@
+/**
+ * @module modules/luceadBidAdapter
+ */
+
 import {ortbConverter} from '../libraries/ortbConverter/converter.js';
-import {loadExternalScript} from '../src/adloader.js';
```
Comment (patmmccann) [inline modules/luceadBidAdapter.js:2]: please drop [from](https://github.com/lucead/Prebid.js/blob/d8b4f90af34af358d763d7b8b0bf080bac6466ae/src/adloader.js#L36) as well

**Review by patmmccann — APPROVED** [review]

**Pull request merged**